moon-store-mac-os/moon-store-mac-os/Router/AppRouter.swift

```diff
@@ -10,17 +10,24 @@ import SwiftUI
 final class AppRouter: ObservableObject {
     @Published var navigationPath: NavigationPath = .init()
     
-    lazy var assosiatedView: some View = {
-        let route: AppTransition = LoginManager.isUserLogged ? .main : .login
+    lazy var associatedView: some View = {
+        let route: AppTransition = authenticationRepository.isLoggedUser
+        ? .main
+        : .login
         return build(for: route)
     }()
     
+    private let authenticationRepository: AuthenticationRepository = .init()
```

Comment:
Move to top

Reply:
`Lets` to top?

Please  be more clare from scratch.